nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -76,6 +76,40 @@ def mutate(self, model):
             target.update_operation(target.operation.type, {**target.operation.parameters, argname: chosen_value})
 
 
+class RepeatMutator(Mutator):
+    def __init__(self, nodes: List[Node]):
```

Comment:
better to explain the meaning of `nodes`

Reply:
ok